src/shogun/mathematics/linalg/LinalgBackendEigen.h

```diff
@@ -764,20 +790,6 @@ namespace shogun
 		template <typename T>
 		void zero_impl(SGMatrix<T>& a) const;
 	};
-
-/*
- * Eigen's symmetric eigensolver uses a slower algorithm in comparison
- * to LAPACK's dsyevr, so if LAPACK is available we use it for float64 type.
- * This should be removed if eventually Eigen will provide a faster
- * symmetric eigensolver (@see
- * http://eigen.tuxfamily.org/bz/show_bug.cgi?id=522).
- */
-#ifdef HAVE_LAPACK
-	template <>
-	void LinalgBackendEigen::eigen_solver_symmetric_impl<float64_t>(
```

Comment:
But now there is no float64_t specialisation? Or is there now a faster symmetric eigensolver provided by Eigen?